File: daily/Arrays/1304.Find-n-unique-integers-sum-up-to-zero.py
from typing import List


class Solution:
    def sumZero(self, n: int) -> List[int]:
        res: List[int] = []
        if n % 2 == 1:
            res.append(0)
        for i in range(1, n // 2 + 1):
            res.extend([-i, i])
        return res

File: daily/Arrays/1304.Find-n-unique-integers-sum-up-to-zero.go
package daily

func sumZero(n int) []int {
	var res []int

	if n%2 != 0 {
		res = append(res, 0)
	}

	for i := 1; i <= n/2; i++ {
		res = append(res, -i, i)
	}

	return res
}
